redis/server.cc

```diff
@@ -163,15 +180,12 @@ future<> redis_server::connection::process()
             f.get();
         }
         catch (redis_exception& e) {
-        logging.info(" redis exception = {} ", e.what());
```

Comment:
You can drop these lines from the original patch - you don't need to add them in patch 2 and drop them in patch 3.

Reply:
Got it.